src/daml/Daml/Triggers/TradingSuccessTrigger.daml

```diff
@@ -0,0 +1,21 @@
+module Daml.Triggers.TradingSuccessTrigger where
+
+import Daml.Trigger
+import Daml.Trading.Reward.TradingPoint
+import DA.Foldable (forA_)
+
+
+issueTradingPointTrigger: Trigger ()
+issueTradingPointTrigger = Trigger with
+  initialize = pure ()
+  updateState = \_ -> pure ()
+  rule = \p -> do
+    debug ("triggered")
+    tradeData <- query @TradeSuccess
+    forA_ tradeData (\tradeSuccessdata -> do
```

Comment:
check if p == admin of tradeSuccess first

Reply:
Done